docs/02-app/01-building-your-application/03-rendering/02-client-components.mdx

```diff
@@ -103,4 +103,4 @@ This means the Client Component JavaScript bundle is downloaded and parsed. Once
 
 Sometimes, after you've declared the `"use client"` boundary, you may want to go back to the server environment. For example, you may want to reduce the client bundle size, fetch data on the server, or use an API that is only available on the server.
 
-You can keep code on the server even though it's theoretically nested inside Client Components by interleaving Client and Server Components and Server Actions. See the [Composition Patterns](/docs/app/building-your-application/rendering/composition-patterns) page for more information.
+You can keep code on the server even though it's theoretically nested inside Client Components by interleaving Client and Server Components and [Server Actions](/docs/app/building-your-application/data-fetching/forms-and-mutations). See the [Composition Patterns](/docs/app/building-your-application/rendering/composition-patterns) page for more information.
```

Comment:
Maybe we could go with "mixing" instead of "interleaving" here? Just for absolute clarity.
```suggestion
You can keep code on the server even though it's theoretically nested inside Client Components by mixing Client and Server Components and [Server Actions](/docs/app/building-your-application/data-fetching/forms-and-mutations). See the [Composition Patterns](/docs/app/building-your-application/rendering/composition-patterns) page for more information.
```